src/recordlinker/routes/person_router.py

```diff
@@ -75,3 +100,34 @@ def update_person(
 
     person = service.update_person_cluster(session, patients, person, commit=False)
     return schemas.PersonRef(person_reference_id=person.reference_id)
+
+
+@router.post(
+    "/{person_reference_id}/merge",
+    summary="Merge Person clusters",
+    status_code=fastapi.status.HTTP_200_OK,
+)
+def merge_person_clusters(
+    person_reference_id: uuid.UUID,
+    data: typing.Annotated[schemas.PersonRefs, fastapi.Body()],
+    session: orm.Session = fastapi.Depends(get_session),
+) -> schemas.PersonRef:
+    """
+    Merges Person cluster(s) into the Person cluster referenced by `person_reference_id`.
+    """
+    # Get the person_reference_id the person clusters will be merged into
```

Comment:
Nit: should be getting the Person itself, since we pass in the person_reference_id